src/mol-io/common/utf8.ts

```diff
@@ -106,6 +110,15 @@ export function utf8Read(data: Uint8Array, offset: number, length: number) {
     }
 }
 
+/** Decode UTF8 data, potentially exceeding MAX_STRING_LENGTH. Return as primitive `string` if possible; or as `ChunkedBigString` if the result is longer than MAX_STRING_LENGTH. */
+export function utf8ReadLong(data: Uint8Array, offset: number = 0, length: number = data.length): StringLike {
```

Comment:
ha, TIL you can reference args in defaults

Reply:
I was also surprised.
Nice feature.